app/src/main/java/de/westnordost/streetcomplete/quests/building_levels/AddBuildingLevels.kt

```diff
@@ -17,6 +17,7 @@ class AddBuildingLevels : OsmFilterQuestType<BuildingLevelsAnswer>() {
                or !roof:levels and roof:shape and roof:shape != flat
            )
            and !building:min_level
+           and building !~ no|houseboat
```

Comment:
Is this line needed? Few lines above it checks only for `building ~ ${BUILDINGS_WITH_LEVELS.joinToString("|")}` and it doesn't seem to include those two:
https://github.com/streetcomplete/StreetComplete/blob/05ef7ce4564e8eba42220ae02e38f87f0e7b6aa5/app/src/main/java/de/westnordost/streetcomplete/osm/BuildingLevels.kt#L3-L9

Are you being asked for building levels on houseboats?

Reply:
There are 17.5k `building=houseboat` worldwide and none of them have a `levels`-tag. I guess SC isn't currently asking for levels on houseboats.

Reply:
Gna. You both are right. 
I was asked about roof shape and 
wanted to go the extra mile and fix the other building quest, too :-)